src/DynamicData/Cache/ObservableCacheEx.cs

```diff
@@ -3316,6 +3316,26 @@ public static IObservable<T> FinallySafe<T>(this IObservable<T> source, Action f
         return source.Where(changes => changes.Count != 0);
     }
 
+    /// <summary>
+    /// Converts the changesets so that it only contains items of the given type.
```

Comment:
Suggest "Filters an observable collection to only include items of a specific type.", or something similar, as the description here.